### ISSUE TITLE
Total Damage Rounding Inaccuracy

### ISSUE THREAD

**Issue by Araeos** [Openarl/PathOfBuilding#951]
The "Total Damage" multiplicator seems to be rounded to the nearest full percent, meaning the calculated damage for skills is inaccurate.
## Example
Skills: Frostbolt lvl 16, Spell Totem lvl 16, Controlled Destruction lvl 14
The Spell Totem reduces Total Damage by 28% and Controlled Destruction increases Total Damage by 38%.
This should result in (100%-28%)*(100%+38%)=99.36% Total Damage (Should be shown as Total Damage -0.64%), while Path of Building shows actually -1% Total Damage.

Path of Building also uses the rounded value for further damage calculations.

**Comment by Wires77** [comment]
Have you tested this on an actual character?  PoE rounds to the nearest whole number at least visually, it may not round behind the scenes, but it's nearly impossible to tell at the level of change we're talking about.

**Comment by Araeos** [comment]
Yes. To expand on this, I only found this through the difference between the display in-game and in Poe. The above calculation accounts for the observed differences. Though I have tested this only on the day I submitted this report as I stopped playing this build.
